native/core/src/events/news.rs

```diff
@@ -101,50 +98,6 @@ pub struct NewsSlide {
     pub references: Vec<ObjRef>,
 }
 
-impl NewsSlide {
-    pub fn new_text(body: String) -> Self {
-        NewsSlide {
-            content: NewsContent::Text(TextMessageEventContent::markdown(body)),
-            references: vec![],
-        }
-    }
-
-    pub fn new_image(body: String, url: OwnedMxcUri) -> Self {
-        NewsSlide {
-            content: NewsContent::Image(ImageMessageEventContent::plain(body, url)),
-            references: vec![],
-        }
-    }
-
-    pub fn new_audio(body: String, url: OwnedMxcUri) -> Self {
-        NewsSlide {
-            content: NewsContent::Audio(AudioMessageEventContent::plain(body, url)),
-            references: vec![],
-        }
-    }
-
-    pub fn new_video(body: String, url: OwnedMxcUri) -> Self {
-        NewsSlide {
-            content: NewsContent::Video(VideoMessageEventContent::plain(body, url)),
-            references: vec![],
-        }
-    }
-
-    pub fn new_file(body: String, url: OwnedMxcUri) -> Self {
-        NewsSlide {
-            content: NewsContent::File(FileMessageEventContent::plain(body, url)),
-            references: vec![],
-        }
-    }
-
-    pub fn new_location(body: String, geo_uri: String) -> Self {
-        NewsSlide {
-            content: NewsContent::Location(LocationMessageEventContent::new(body, geo_uri)),
-            references: vec![],
-        }
-    }
-}
```

Comment:
as said above, I do not agree that this should all be moved from here to the `api`-crate. Quiet the opposite, I think it would be better to change the interface to taking the specific event-content as the param but keep this API.

Like:
```rust
  pub fn new_text(content: TextMessageEventContent) -> Self {
        NewsSlide {
            content: NewsContent::Text(content),
            references: vec![],
        }
    }

    pub fn new_image(content: ImageMessageEventContent) -> Self {
        NewsSlide {
            content: NewsContent::Image(content),
            references: vec![],
        }
    }
```